### ISSUE TITLE
display mbtiles file name in addition to layer name?

### ISSUE THREAD

**Issue by springmeyer** [mapbox/MapBoxiPad#91]
This is a very minor thing, and feel free to close if you've already decided against this. 

But basically the request would be to display the mbtiles filename is very light, small grey text below the layername in the layer list. This way, if you have loaded an mbtiles filename with "abcdefg.mbtiles" but the layer inside (which you did not create yourself so you have no idea what is inside) is called "foo", you might have some idea where they layer is that you loaded.

Basically my concern is that as people start sharing more mbtiles between them, if layers have similar names, like "points", or "overlay" then there is no way you can distinguish or even guess which layer represents the mbtiles file you loaded.
